shared/actions/config.js

```diff
@@ -96,6 +96,7 @@ const _retryBootstrap = () =>
 // TODO: It's unfortunate that we have these globals. Ideally,
 // bootstrap would be a method on an object.
 let bootstrapSetup = false
+let didInitialNav = false
```

Comment:
Perhaps I should put this (and `bootstrapSetup`) in `state.config`?

Reply:
Let's change this very little. This code needs to be revisited holistically